server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersHelpers.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Description: Helper functions for count advertisers cloud functions.
+ * Author: Robert Marcus
+ * Date: July 14, 2020
+ */
+
+const { assert } = require('../test/testConfig');
```

Comment:
This isn't a test so you shouldn't be using assert.  If there's an error, throw an exception.